strr-examiner-web/app/components/DecisionPanel.vue

```diff
@@ -56,23 +56,23 @@ const decisionButtons = [
     action: ApplicationActionsE.APPROVE,
     label: t('btn.approve'),
     color: 'green',
-    activeClass: 'bg-green-100',
+    selected: 'bg-str-bgGreen text-str-textGray hover:bg-white',
```

Comment:
In terms of readability, does a name such as `activeStyle` instead of `selected` be more clear?

Reply:
Naming is hard, I was going for a name that matches the action. ActiveStyle sounds good to me.